### ISSUE TITLE
Crash when start publish iPod Touch 5gen 

### ISSUE THREAD

**Issue by shogo4405** [shogo4405/lf.swift#109]
* XCode 8+
* 0.5.0
* iOS 8.1

Console log
```txt
fatal error: storeBytes to misaligned raw pointer
```

**Comment by shogo4405** [comment]
I think... xcode7 build works fine.

**Comment by shogo4405** [comment]
```txt 
extension ExpressibleByIntegerLiteral {
    var bytes:[UInt8] {
        var data:[UInt8] = [UInt8](repeating: 0, count: MemoryLayout<`Self`>.size)
        data.withUnsafeMutableBufferPointer {
            UnsafeMutableRawPointer($0.baseAddress!).storeBytes(of: self, as: Self.self) // crash here
        }
        return data
    }
```

**Comment by dmdque** [comment]
I'm having the same issue on a 1st gen iPad mini with iOS 9.3.3.

**Comment by shogo4405** [comment]
32bits device

**Comment by 0x384c0** [comment]
same issue on ios simulator 10

**Comment by shogo4405** [comment]
Probably. I fixed it. https://github.com/shogo4405/lf.swift/commit/95aef7f5454361c83bbc17daf0f33da2bc4448f4
I will release 0.5.1 30 Sep :)

**Comment by shogo4405** [comment]
Released 0.5.1
